cluster/common.sh

```diff
@@ -128,8 +128,16 @@ function create-kubeconfig() {
 # Assumed vars:
 #   KUBECONFIG
 #   CONTEXT
+#
+# To explicitly name the context being removed, use OVERRIDE_CONTEXT
 function clear-kubeconfig() {
   export KUBECONFIG=${KUBECONFIG:-$DEFAULT_KUBECONFIG}
+  OVERRIDE_CONTEXT=${OVERRIDE_CONTEXT:-}
```

Comment:
This is an independent change. Without this change, running `hack/e2e.go --up` followed by `hack/e2e.go --down` would leave contexts in kubeconfig that can affect later runs. This change ensures that kubeconfig is cleared correctly.